modMcf/src/org/aion/mcf/config/CfgSync.java

```diff
@@ -72,6 +79,36 @@ public void fromXML(final XMLStreamReader sr) throws XMLStreamException {
         }
     }
 
+    private static void parseSelectedStats(List<StatsType> showStatistics, String readValue) {
```

Comment:
minor/optional suggestion: I think things would be simpler if showStatistics was a Set instead of a List

Reply:
I agree with this